### DISCUSSION TITLE
Mgmt subnet - a /21 or larger subnet is recommended

### DISCUSSION THREAD

**Discussion by Martinagsr** [aws-samples/aws-secure-environment-accelerator#829] [Q&A] [answered]
Mgmt subnet - "a /21 or larger subnet is recommended", this configuration may represent more than 30 000 IP adresses for the workloads VPC if we have three availibilty zones.

Do you have a more explicit use of the Mgmt subnet? Do AWS recommand to have two interfaces on each EC2?

**Comment by Brian969** [comment, marked as answer]
- Multiple interfaces are generally not relevant in the cloud, and in most cases I discourage their use.
- The Mgmt subnet is actually not highly used and included as a legacy holdover for corner cases that don't fit with some of the modern cloud approaches.  As the architecture you are looking at leverages a shared VPC architecture and because of the dynamic nature of auto-scaling and potentially broad use of IP's newer technologies in the cloud, we simply wanted to ensure customers didn't run out of IP addresses.  I'm actually not against totally removing the subnet.  Some customers are putting Amazon Workspaces for their Developers into that subnet - a seemingly good use.  We highly discourage bastion or jump hosts, but if you had them you'd also put them into this subnet.
- CIDR range sizing is also relative to the size of your organization - while the examples all use a class B per VPC - my smaller customers are allocating 1/2 class B's per VPC (/17).  I don't like going much smaller in a shared VPC architecture (which saves you from silo'd IP pools and stranded IP's).
